src/pset/map/output.rs

```diff
@@ -117,13 +166,46 @@ impl Output{
         rv
     }
 
+    /// Create a output from unblinded txout
+    /// This is to be used when the txout is not blinded. This sets
+    /// the blinding key from the txout nonce
```

Comment:
In what contexts is this function actually used? Its purpose seems to be to deal with a transaction which has been partially created using the old Elements rawtransactions API, in which case it should have a more obscure name (if it should be used at all).

Reply:
It is how it is currently implemented in el`ementsd`. There is some RPC that creates a psbt from raw transaction and we need to mimic it's behaviour.